src/librustdoc/html/static/main.js

```diff
@@ -890,6 +890,22 @@
         return "\u2212"; // "\u2212" is '−' minus sign
     }
 
+    $(".search-input").on("keyup",function() {
```

Comment:
Please move content of the handler into the already existing handler (near the top of `function startSearch`).